impeller/display_list/skia_conversions.cc

```diff
@@ -9,6 +9,20 @@
 namespace impeller {
 namespace skia_conversions {
 
+bool IsNearlySimpleRRect(const SkRRect& rr) {
+  auto [a, b] = rr.radii(SkRRect::kUpperLeft_Corner);
```

Comment:
Not a huge deal, but you can just check `type()` and `getSimpleRadii()`.

Reply:
If I don't know that the rrect is simple, then I still need to check that both of the top left corner radii are equal, so I end up calling this method anyway.

Reply:
dl_dispatcher could check isSimple first before calling this method. This method (when fixed to the right definition) will always return true if the SkRRect method returns true, but the Skia method is faster since it just checks the internal type.

Only if the simple test returns false is the more careful test needed...